src/map/clif.hpp

```diff
@@ -1187,4 +1187,7 @@ void clif_inventory_expansion_info( struct map_session_data* sd );
 void clif_barter_open( struct map_session_data& sd, struct npc_data& nd );
 void clif_barter_extended_open( struct map_session_data& sd, struct npc_data& nd );
 
+void clif_summon_init(struct mob_data* md);
+void clif_summon_hp_bar(struct mob_data* md);
```

Comment:
You could use mob_data& for both cases to remove nullpointer possibility.

Reply:
ok. I already fix its. please recheck them.